src/utils/get-own-property-descriptors.js

```diff
@@ -0,0 +1,7 @@
+export default function getOwnPropertyDescriptors(o) {
+  return Object.getOwnPropertyNames(o).reduce((accumulator, propName) => {
```

Comment:
How is this different from `Object.getOwnPropertyDescriptors`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptors

Reply:
It's not but I couldn't get it to work in typescript. I can use a polyfill now.